src/SeleniumLibrary/keywords/browsermanagement.py

```diff
@@ -238,6 +239,39 @@ def switch_browser(self, index_or_alias):
                    % self.browser.session_id)
 
     @keyword
+    def get_browser_capabilities(self):
+        """Gets current browser capabilities from webdriver.
+
+        Returns a Dictionary with webdriver (browser) desired capabilities attributes.
+        You may also query keys providing a default value when those keys are nonexistent.
+
+        Examples of returned attributes are (in *bold* the common ones):
+        'rotatable','takesScreenshot', 'acceptSslCerts', 'cssSelectorsEnabled', 'javascriptEnabled',
+        'databaseEnabled', 'locationContextEnabled', '*platform*', '*browserName*', '*version*',
+        'nativeEvents', 'applicationCacheEnabled', 'webStorageEnabled', 'handlesAlerts'
+        
+        Note: Firefox/Geckodriver is returning different names for '*platform*' and '*version*'.
+              It returns: '*platformName*' and '*browserVersion*'.
+        
+        Examples:
+        | ${capabilities}= | Get Browser Capabilities                            |                           |
+        | Log              | ${capabilities.browserName} ${capabilities.version} | # Using attributes        |
+        | Log              | Capabilities = ${capabilities}                      |                           |
+        | Log              | ${capabilities['platform']}                         | # Using dictionary        |
+        | Log              | ${capabilities.get('does_not_exist', 'default')}    | # Gets 'default'          |
+        | Log              | ${capabilities['does_not_exist']}                   | # Produces KeyError       |
+        | Log              | ${capabilities.does_not_exist}                      | # Produces AttributeError |
+
+        Reference, see [https://seleniumhq.github.io/selenium/docs/api/py/webdriver/selenium.webdriver.common.desired_capabilities.html|Common Desired Capabilities].
+
+        New in SeleniumLibrary 3.0.
+        """
+
+        capabilities = self.browser.desired_capabilities
```

Comment:
Write return DotDic (self.browser.desired_capabilities)

Reply:
Also remove not required new lines.

Reply:
Done